### ISSUE TITLE
item_data.json

### ISSUE THREAD

**Issue by raphaell-git** [matchatealeaf/albion-discord-bot#1]
Hello, first of all thank you for sharing this bot, it is really interesting and fun to play around with. 

However, I have found one issue that I am not sure how to deal with: I have been noticing that the bot sometimes misses completely when searching for some items, such as the "Adept's Imbued Mace Head", and I realized that this item does not have the correct name in the item_data.json file.

The unique name is "T4_ARTEFACT_2H_MACE_MORGANA", the correct name should be "Adept's Imbued Mace Head", but in item_data.json it is named "Adept's Camlann Mace".

My question is, how was this item_data.json file generated? Can we somehow update/fix it automatically? The same issue happens with all of these "artefact" items, from what I can tell.

**Comment by matchatealeaf** [comment]
Hi, thanks for informing on this.

The item_data.json file comes from the [ao-bin-dumps project](https://github.com/broderickhyman/ao-bin-dumps), and they have kindly formatted the item data into [items.json](https://github.com/broderickhyman/ao-bin-dumps/tree/master/formatted).

I realised that they have [fixed the artefact names back in 27 Jan](https://github.com/broderickhyman/ao-bin-dumps/commit/cb19f107b1d2876265f5870ac5bdcb041c004aab), but I failed to update the .json file on this side. I'm terribly sorry about this.

I have updated the file in my [latest commit](https://github.com/matchatealeaf/albion-discord-bot/commit/b122cd826864fcce9aee335e4c2a6f39b324363e), and it should be working now. Although, the bot that I'm hosting will take some time before I can update it.

Let me know if updating the file doesn't fix the issue for you. Thanks.
